packages/wrangler/src/cloudchamber/deploy.ts

```diff
@@ -17,6 +17,7 @@ export async function maybeBuildContainer(
 	imageTag: string,
 	dryRun: boolean,
 	pathToDocker: string,
+	/** The original (non-redirected) user config path */
```

Comment:
what about renaming the parameter `userConfigPath` to avoid possible future confusion? 🤔